mkt/search/views.py

```diff
@@ -101,6 +101,12 @@ def name_query(q):
     return query.Bool(should=should)
 
 
+def _get_regional_sort(region, field):
+    if region and not region.adolescent:
+        return ['-%s_%s' % (field, region.id)]
+    return ['-%s' % field]
```

Comment:
That wasn't here before, so, before this PR, it was impossible for an adolescent region to get explicit sorting of a query by popularity or trending ?